src/test/miner_tests.cpp

```diff
@@ -247,7 +246,7 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     tx.vin[0].prevout.n = 0;
     tx.vout.resize(1);
     tx.vout[0].nValue = BLOCKSUBSIDY;
-    for (unsigned int i = 0; i < 1001; ++i)
+    for (unsigned int i = 0; i < 2001; ++i)
```

Comment:
This is doubled because GoldCoin has a 2MB block size limit instead of the 1 MB limit of litecoin.